ui/src/app/utils/access-utils.tsx

```diff
@@ -60,8 +60,8 @@ export const redirectToNiH = (): void => {
   AnalyticsTracker.Registration.ERACommons();
   const url = serverConfigStore.get().config.shibbolethUiBaseUrl + '/login?return-url=' +
```

Comment:
FYI: you should be able to just construct this locally and enter it into your browser on staging. Also fine if you want to deploy-ui on staging though, just remember to revert.

Reply:
RAS does not add localhost:4200 into their the prod client id allowlist. I need to deploy-ui on staging using this PR after this week release

Reply:
moot point now, but I meant either locally or manually, generating a URL for **staging**, not localhost; then just pasting into the browser.